contrib/resources/translations/it.lng

```diff
@@ -617,55 +614,6 @@ Usa 'allow' (consenti) o 'block' (blocca) per sovrascrivere la variabile
-  animation=<valore>: Se impostato su 'on' (predefinito), abilita l'animazione
-                      del simbolo di registrazione audio/video. Impostare su
-                      'off' per disabilitare l'animazione; questo è utile se
-                      i caratteri dello schermo producono strani risultati.
-  program=<valore>:   Visualizza il nome del programma in esecuzione.
-                      <valore> può essere uno dei seguenti:
-                        none/off:  Non visualizza il nome del programma.
-                        name:      Nome del programma con estensione file
-                                   (predefinito).
-                        path:      Nome, estensione e percorso completo.
-                        segment:   Visualizza il nome del segmento di memoria
-                                   del programma.
-                        'Titolo':  Nome personalizzato. In alternativa è
-                                   possibile utilizzare il formato "Titolo",
-                                   (Titolo), <Titolo> o [Titolo].
-                      Nota: Con alcuni software (come Windows 3.1x in modalità
-                      avanzata) è impossibile riconoscere il nome o il percorso
-                      completo del programma; in questi casi viene utilizzato
-                      il parametro 'segment'.
-  dosbox=<valore>:    Visualizza 'DOSBox Staging' nella barra del titolo.
-                      <valore> può essere uno dei seguenti:
-                        always:   Visualizza sempre 'DOSBox Staging'.
-                        auto:     Visualizza il valore prefissato solo se
-                                  nessun programma è in esecuzione o se è
-                                  impostato 'program=none' (predefinito).
-  version=<valore>:   Visualizza le informazioni sulla versione di DOSBox.
-                      <valore> può essere uno dei seguenti:
-                         none/off:  Non visualizza la versione di DOSBox
-                                    (predefinito).
-                         simple:    Informazioni di base sulla versione.
-                         detailed:  Visualizza anche la revisione o hash Git,
-                                    se disponibile.
-  cycles=<valore>:    Se impostato su 'on' (predefinito), mostra l'impostazione
-                      dei cicli della CPU. Impostare su 'off' per disabilitare
-                      la visualizzazione dell'impostazione dei cicli.
-  mouse=<valore>:     Livello di verbosità dei messaggi di cattura del mouse:
-                        none/off:  Non visualizza alcun messaggio.
-                        short:     Visualizza solo il messaggio di cattura
-                                   del mouse.
-                        full:      Visualizza informazioni dettagliate su come
-                                   catturare o rilasciare il cursore
-                                   (predefinito).
-.
 :CONFIG_LANGUAGE
```

Comment:
Are you sure you wanted to delete this?

Reply:
No, that's why I put it back in draft 😁 I think I made a mess with rebasing

Reply:
Sure, no rush :)